### DISCUSSION TITLE
How does MAUI release cycle look like?

### DISCUSSION THREAD

**Discussion by lubiepomaranczki** [dotnet/maui#19653] [General]
Hello there,
what is the MAUI release cycle? When can we expect service releases? 🤔 

The team I'm working in is blocked by [this issue](https://github.com/dotnet/maui/issues/17706).

@Redth wrote in this [comment](https://github.com/dotnet/maui/issues/17706#issuecomment-1808741944) that this will be part of the service build but I can't find it anywhere.

It would be good to know the cycle of service releases, so people can decide whether to create a workaround or wait for the build. It will increase developer experience and help us plan our work.

When is the next service release scheduled?

**Comment by drasticactions** [comment]
This is a question, not really an "bug" or an issue, but it's a good conversation to have. I'm moving this to there so ya'll can talk about it.

**Reply by lubiepomaranczki** [reply]
sorry, wasn't aware of such GitHub feature as discussions! Thanks!

**Comment by phillippschmedt** [comment]
@lubiepomaranczki I just wanted to let you know that we are facing the same issue (https://github.com/dotnet/maui/issues/17706) and are also waiting for the SR1. I have been watching https://github.com/dotnet/maui/milestones and specifically https://github.com/dotnet/maui/milestones/.NET%208%20SR1 to check the progress. All SR1 issues are closed.

**Reply by jurganson** [reply]
+1

**Comment by AlleSchonWeg** [comment]
MAUI is part of .NET:
https://github.com/dotnet/maui/wiki

> .NET MAUI ships with each .NET release on a monthly cadence.

The updates are released 

> on the Microsoft "Patch Tuesday" (second Tuesday of each month)

Source:  https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#servicing

You could also try the Maui Nightly Builds: https://github.com/dotnet/maui/wiki/Nightly-Builds
This builds should include your blocking issue.

**Reply by lubiepomaranczki** [reply]
thanks for the answer! We are fully aware of those. 

Unfortunately, to my knowledge, there was no release in December - "Patch Tuesday" didn't happen since .NET 8 was released.

About nightly builds, in my opinion, that is not production ready solution. I won't risk releasing my app on that. 

I still believe some schedule should be helpful. Especially now, when there are many things that don't work.

**Reply by Stedy59** [reply]
A bit confused here... I've never seen Maui updated via "Patch Tuesday",  only via Visual Studio releases.

**Comment by TheFerrango** [comment]
We are also stuck on this issue and have resort to keep using maui 8.0.0-nightly until it's fixed in the public release
